### ISSUE TITLE
Error installing Uintra 2.0.0

### ISSUE THREAD

**Issue by jkoletar** [Uintra/Uintra#120]
I was excited to see Uintra 2.0.0 hit NuGet, but I am seeing the same issue after installation that I saw with the 2.0.0 beta.  I get errors in the index.cshtml as the loaded Model is not populated - all the Model properties are null.

I followed the same steps with Uintra 1.4.0 and had no issues.

1. Create a new ASP.NET Web Application in Visual Studio.
2. Open up NuGet Package Manager
3. Add Uintra 2.0.0
4. Start project with debugging
5. Point Umbraco to empty SQL Server database. 
6. Decline Starter website.
7. Get error trying to load page

![image](https://user-images.githubusercontent.com/2780829/96467265-5ae96300-11f0-11eb-90e4-11e805102ed6.png)

Looking in the BackOffice I see that there is no template for the Home page.  After generating and adding the template, I get the same error.

Is there something I am missing?

**Comment by 1nsolenT** [comment]
Hi @jkoletar , you should follow these steps: 
1. Use IIS instead of IISExpress. 
2. Add your domain (which was added in IIS) to "Cultures and Hostnames". 
3. Run uSync (it may take a few minutes).
4. Change <add key="Search.IndexName" value="[your index name]" /> in web.config

**Comment by jkoletar** [comment]
Thanks for the tips @1nsolenT.

I ran into issues installing Uintra 2.0.0 on IIS.  The uninitialized site came up and after entering the SQL configuration info the Umbraco install seemed to run fine, but threw an error on Uintra install.  I tried installing Uintra 2.0.0 again in IISExpress.  After setting the domain and running uSync,  the site comes up and works in IISExpress.

Here was the error I got with the install on IIS:
![image](https://user-images.githubusercontent.com/2780829/96608289-d1549680-12be-11eb-89ea-ac896d4170de.png)

Looking at the database after the failed install the Umbraco tables are there, as well as ExternalLogins and SkybrudRedirects, but there are no Uintra tables.
